ksh93/PKGBUILD

```diff
@@ -0,0 +1,124 @@
+# This took a lot of trial and error. There possibly should be patches from the linked gisburn repo and the ones from debian and red hat.
+#
+pkgname=ksh93
+pkgver=1.0.10
+pkgrel=1
+pkgdesc="The Original ATT Korn Shell, Now Maintained By The Community."
+arch=('x86_64')
+url="https://www.github.com/ksh93/ksh"
+license=('spdx:EPL-2.0')
+
+# These makedeps were suggested by 'gisburn' in a ksh93 issue.
+makedepends=(
+	'gcc'
+	'gdb'
+	'make'
+	'gettext'
+	'gettext-devel'
+	'flex'
+	'bison'
+	'unzip'
+	'pax'
+	'tar'
+	'libiconv-devel'
+	'ncurses-devel'
+	'gmp-devel'
+	'mpfr-devel'
+	'mpc-devel'
+	'isl-devel'
+	'procps-ng'
+)
+# There exists already a package that provides ksh.exe, mksh, so I will set this to be a conflict.
+# Unless there can be a convincing reason to let both packages be installed.
+conflicts=('mksh')
+
+source=("${pkgname}-${pkgver}.zip"::https://github.com/ksh93/ksh/archive/refs/tags/v1.0.10.zip
+	'ksh.kshrc')
+sha256sums=('ab412118ce9527c6d805335080593fa874a0b859e9229c172f3fcf55d26f92e9'
+	'SKIP')
+
+# This is necessary
+export PATH="/bin:/usr/bin:$gccdir:$gccdir/include:/usr/include"
+
+# Things might get a bit messary, prepare yourselves.
+# I really should use gisburn's kshrc default config.
+build() {
+	cd "${srcdir}/ksh-${pkgver}"
+	
+	export CYGWIN_CHOST="${CHOST/-msys/-cygwin}"
```

Comment:
this is used in other PKGBUILDs as an argument to configure.  I don't see you using it anywhere here, and I doubt ksh93's build scripts expect it to be set.  Maybe you could try `local MSYSTEM=CYGWIN` and see if anything useful changes?